analysis_options.yaml

```diff
@@ -9,6 +9,9 @@
 # packages, and plugins designed to encourage good coding practices.
 include: package:flutter_lints/flutter.yaml
 
+analyzer:
+  exclude: [plugin/**]
+
```

Comment:
Note to self: should be folded back into rid

Reply:
As well as deleting `plugin/analysis_options.yaml`